doc/users/dflt_style_changes.rst

```diff
@@ -560,6 +560,13 @@ or by setting::
 
 in your :file:`matplotlibrc` file.
 
+``hexbin``
+----------
+
+The default value of the ``linecolor`` kwarg for `~matplotlib.Axes.hexbin` has
+changed from ``'none'`` to ``'face'``. If 'none' is now supplied, no line edges
+are drawn around the hexagons.
```

Comment:
This should note that the default _behavior_ has not changed (just the meaning of `'none'`).